### DISCUSSION TITLE
Upgrading Grafana to 8.3.2 results in parameter errors

### DISCUSSION THREAD

**Discussion by alonehat** [grafana/grafana#43254] [Dashboards, Panels & Visualizations]
**Describe the bug**        

> I used version 8.1.1 and used the 7362 dashboard to monitor the mysql database, when I noticed that there was a serious security vulnerability in the version below 8.3.x, I upgraded the version with rpm, but when I opened the dashboard, I found that the dashboard using the $interval parameter was all reported errors.
The following are the system configuration instructions before and after the upgrade:

- Before:
    - os: Centos 7.4
    - grafana: grafana-8.1.1-1.x86_64.rpm
    - mysql: 5.7.36
    - mysqld_exporter: mysqld_exporter-0.10.0.linux-amd64
    - prometheus: prometheus-2.7.2.linux-amd64
- Now:
    - os: Centos 7.4
    - grafana: grafana-8.3.2-1.x86_64.rpm
    - mysql: 5.7.36
    - mysqld_exporter: mysqld_exporter-0.10.0.linux-amd64
    - prometheus: prometheus-2.7.2.linux-amd64

**Expected behavior**

> Because so many monitoring dashboards are configured, it is desirable to have a way to make them work without modifying existing dashboards.  
> **If I modify $interval to a specific numeric value, they work correctly**

**Screenshots**
![image](https://user-images.githubusercontent.com/33916962/145698944-f3d383ab-377d-4023-ac01-b7df394b36a6.png)

![image](https://user-images.githubusercontent.com/33916962/145698951-a9279a27-9920-45fe-8c08-97f4f2a88304.png)

**Network data**

>t=2021-12-12T11:21:49+0800 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/ds/query status=500 remote_addr=192.168.0.102 time_ms=0 size=34 referer="http://192.168.0.104:3000/d/wdP2gQh7k/mysql-overview?from=now-30m&to=now&orgId=1&refresh=1m"
t=2021-12-12T11:21:49+0800 lvl=eror msg="Metric request error" logger=context userId=1 orgId=1 uname=admin error="failed to query data: time: invalid duration \"$interval\"" remote_addr=192.168.0.102
t=2021-12-12T11:21:49+0800 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/ds/query status=500 remote_addr=192.168.0.102 time_ms=0 size=34 referer="http://192.168.0.104:3000/d/wdP2gQh7k/mysql-overview?from=now-30m&to=now&orgId=1&refresh=1m"
t=2021-12-12T11:21:49+0800 lvl=eror msg="Failed to rotate token" logger=context userId=1 orgId=1 uname=admin error="Rolling back transaction due to error failed: cannot rollback - no transaction is active: context canceled"
t=2021-12-12T11:22:22+0800 lvl=eror msg="Metric request error" logger=context userId=1 orgId=1 uname=admin error="failed to query data: time: invalid duration \"$interval\"" remote_addr=192.168.0.102
t=2021-12-12T11:22:22+0800 lvl=eror msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/ds/query status=500 remote_addr=192.168.0.102 time_ms=1 size=34 referer="http://192.168.0.104:3000/d/wdP2gQh7k/mysql-overview?from=now-30m&to=now&orgId=1&editPanel=92"

**Software versions**
> ditto

Thanks for checking in.

**Comment by alonehat** [comment]
how to fix it?

**Reply by mckn** [reply]
Can you please create an issue so the proper team can investigate this further. I don't think they have noticed this since we expect bugs/errors to be reported as issues.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
